addons/nuxeo-ai-aws-core/src/main/java/org/nuxeo/ai/enrichment/DetectCelebritiesEnrichmentService.java

```diff
@@ -58,21 +65,58 @@ public void init(EnrichmentDescriptor descriptor) {
     }
 
     @Override
-    public Collection<EnrichmentMetadata> enrich(BlobTextFromDocument blobTextFromDoc) {
+    public Collection<EnrichmentMetadata> enrich(BlobTextFromDocument doc) {
+        RekognitionService rs = Framework.getService(RekognitionService.class);
+        if (doc.getFacets().contains(VIDEO_FACET)) {
```

Comment:
I don't really understand these changes to `EnrichmentServices`.  It seems that if the doc has `VIDEO_FACET` you assume the blob to be sent is a video.  I expected either new classes or some configurable flag on the existing services.